code/android-core-library/src/phone/java/com/adobe/marketing/mobile/MobileCore.java

```diff
@@ -991,4 +991,10 @@ public static boolean dispatchResponseEvent(@NonNull final Event responseEvent,
         return true;
     }
 
+    @VisibleForTesting
+    public static void resetSDK() {
```

Comment:
A public method visible for testing is a bit odd for an implementer using the SDK. What do you think about adding a test utility helper in the same package?